### PR TITLE
[Components with more than 1 service] LPS-179214 Refactor ScreenNavigationCategory components from USM

### PR DESCRIPTION
**Pull request by MrJohn1911**

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:usm

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions.

**Review by rafaprax — APPROVED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions.

**Review by rafaprax — APPROVED** [review]

**Comment by MrJohn1911** [comment]
@rafaprax can you review this? I just changed the commit message.

**Comment by tinatian** [comment]
@rafaprax The ticket number in commit message is LPS-178898, which the branch name is LPS-179214, which one should be the right ticket for this change?

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/frontend/taglib/servlet/taglib/AccountRoleDefineGroupScopePermissionsScreenNavigationEntry.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		ResourceBundle resourceBundle = ResourceBundleUtil.getBundle(
+			"content.Language", locale, getClass());
+
+		return new AggregateResourceBundle(
+			resourceBundle, portal.getResourceBundle(locale));
+	}
+
+	@Reference
+	private Language _language;
+
```
Comment (tinatian) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/frontend/taglib/servlet/taglib/AccountRoleDefineGroupScopePermissionsScreenNavigationEntry.java:82]: @MrJohn1911 @rafaprax This part should be extracted to base class, as it was not there before, it should be something in common.

Tina

**Comment by tinatian** [comment]
@MrJohn1911 @rafaprax Please see my comment and resend.

Thanks
Tina

**Pull request closed without merging**